config.ru

```diff
@@ -26,10 +26,9 @@ use Rack::Rewrite do
   }
   r301 "/changelog/", "/changelog"
 
-  r301 %r{.*}, "https://#{ENV["CANONICAL_HOST"]}/cli$&", if: proc { |rack_env|
+  r301 %r{.*}, "https://#{ENV["CANONICAL_HOST"]}/platform/cli/start$&", if: proc { |rack_env|
     ["cli.scalingo.com"].include?(rack_env["SERVER_NAME"])
   }
-  r301 "/cli/", "/cli"
```

Comment:
You're right, it may be a good idea to keep the redirect for SEO purpose, it won't hurt anyway.
I will set a redirect for `/cli/` to `/platform/cli/start`. It will avoid having 2 redirects one after the other `/cli/` -> `/cli` follow by `/cli` to `/platform/cli/start`.